### DISCUSSION TITLE
Having A- tag label for scope definition?

### DISCUSSION THREAD

**Discussion by kwonoj** [swc-project/swc#3177] [General]
Related : 

`meta` issues like https://github.com/swc-project/swc/issues/2934 / https://github.com/swc-project/swc/issues/3167 .

I'm curious if we could have another category of label `A` (for Area) to label issues / changes to specifica code or behavior areas we'd like to group. Borrwoed idea from https://github.com/servo/servo/wiki/Tag-label-names-and-definitions .

**Comment by kwonoj** [comment]
Examples like

```
A-cli
A-plugin
A-minifier
A-bundler
A-transform
```

...etcs?

**Reply by kdy1** [reply]
I think this is good to do

**Reply by kwonoj** [reply]
Cool, I created first label `A-plugin` for the tracking plugin work. We can add more things later.

**Comment by kwonoj** [comment]
It is agreed and already applied, wrapping up discussion.
